server/chat-plugins/helptickets.ts

```diff
@@ -619,6 +620,15 @@ const ticketPages: {[k: string]: string} = {
 	confirmroomhelp: `Call a Global Staff member to help`,
 	confirmother: `Call a Global Staff member`,
 };
+const cheatingScenarios = [
+	[`My opponent's Pokemon used moves it couldn't learn`, `It was probably a disguised Zoroark (<psicon pokemon = "zoroark">), which has the ability <a href = "https://dex.pokemonshowdown.com/abilities/illusion">Illusion</a>. This happens often in Random Battles!`],
```

Comment:
```suggestion
	[`My opponent's Pokemon used moves it couldn't learn`, `It was probably a disguised Zoroark (<psicon pokemon="zoroark" />), which has the ability <a href="https://dex.pokemonshowdown.com/abilities/illusion">Illusion</a>. This happens often in Random Battles!`],
```
Some vague mention of OMs might help?

Reply:
We only ever get "omg they are hacking!" from Zoroark in randbats (or at least, 95%+), because no Team Preview makes Illusion really annoying.